products/dotnet/get-started.adoc

```diff
@@ -204,4 +206,4 @@ Uninstalling the `rh-dotnetcore10-runtime` package will cause the dependent pack
 ----
 . *Some .NET/C# code/examples I’ve tried don’t work with .NET Core.*
 +
-.NET Core 1.0 is a new version of the .NET framework that is incompatible with the previous .NET 2.x, 3.x and 4.x series frameworks. There is a large amount of code written for .NET that will not run without modification on .NET Core.
+{empty}.NET Core 1.0 is a new version of the .NET framework that is incompatible with the previous .NET 2.x, 3.x and 4.x series frameworks. There is a large amount of code written for .NET that will not run without modification on .NET Core.
```

Comment:
At the top of the page, can we make the following changes to the banner sentence?
* change Enterprise to enterprise
* change Distribution to distribution

Build and run cross-platform .NET applications on the world's number one enterprise-ready Linux distribution, Red Hat Enterprise Linux.